sdk/typescript/dev/src/ci.ts

```diff
@@ -0,0 +1,57 @@
+import { Container, dag, func, object } from "@dagger.io/dagger"
+
+@object
+export class CI {
+  ctr: Container
+
+  constructor(ctr: Container) {
+    this.ctr = ctr
+  }
+
+  /**
+   * Run the CI (lint, test, build).
+   */
+  @func
+  async run(): Promise<void> {
+    await Promise.all([
+      await this.lint(),
+      await this.test(),
+      await this.build().stdout(),
```

Comment:
`sync` instead of `stdout` since you're not using that output.

Reply:
I removed CI in [4474436](https://github.com/dagger/dagger/pull/6522/commits/4474436f1e8112cf2e70b5f2e8ea0326b27ca936) in favor of top level functions